terraform/cloudwatch.tf.json.template.py

```diff
@@ -1,34 +1,96 @@
+import json
+
 from azul import (
     config,
 )
 from azul.terraform import (
     emit_tf,
 )
 
-emit_tf(None if config.disable_monitoring else {
-    "resource": [
+
+def dashboard_body() -> str:
+    with open(config.cloudwatch_dashboard_template) as f:
+        body = json.load(f)
+    body = json.dumps(body)
```

Comment:
Why do you send this through `load` and `dumps`? If there's a valid reason, a comment should explain it.

Reply:
Parsing the JSON only to then convert it back to a string has two benefits, validating the JSON of the template before deployment and stripping out all excess whitespace. I've added a comment to explain.